infrastructure/src/main/java/org/corfudb/infrastructure/LogUnitServerCache.java

```diff
@@ -61,17 +58,6 @@ public LogUnitServerCache(LogUnitServerConfig config, StreamLog streamLog) {
 
         MeterRegistryProvider.getInstance().ifPresent(registry ->
                 CaffeineCacheMetrics.monitor(registry, dataCache, "logunit.read_cache"));
-        hitRatio = MeterRegistryProvider.getInstance().map(registry ->
-                Gauge.builder(hitRatioName,
-                dataCache, cache -> cache.stats().hitRate()).register(registry));
-        loadTime = MeterRegistryProvider.getInstance().map(registry ->
-                Gauge.builder(loadTimeName,
-                        dataCache, cache -> cache.stats().totalLoadTime())
-                        .register(registry));
-        weight = MeterRegistryProvider.getInstance().map(registry ->
-                Gauge.builder(weightName,
-                        dataCache, cache -> cache.stats().evictionWeight())
-                        .register(registry));
```

Comment:
Should we keep this part? They are assigned and looks like never used, but I think the metric register will expose these gauges when it outputs metrics.

Reply:
Done, Thank you.